### PR TITLE
Renamed the shopkeepers to be CS professors.

### PR DESCRIPTION
**Pull request by dgonyeo**
Names taken from:
http://www.cs.rit.edu/people/faculty

Note that this will result in running out of unique shopkeepers faster.
I don't think this will have an adverse effect on gameplay.

### CODE REVIEW

**Comment by dgonyeo** [comment]
I debated just presenting the last names instead of the full names, and which professors run which shops was mostly random. If anyone has strong opinions on either I'll be happy to change it.

**Comment by mansam** [comment]
Maybe mix in the names of professors from other computing majors so that there's a little more variety? Full names feels sorta stilted, but I don't feel particularly strongly about it.

**Comment by ryansb** [comment]
+1 to adding profs from other departments in GCCIS. 

Additionally, it would be excellent if the liquor store shopkeepers were reslife employees.

**Comment by dgonyeo** [comment]
I added some other departments in, and switched to last names. Res Life now runs all the liquor stores, but I decided to use full names for res life people (I know who Tarra is, but who the fuck is Ognissanti?).

How's that look? Thoughts on removing some people from the res life section?

**Comment by ryansb** [comment]
Looks great :100:

**Comment by mansam** [comment]
+1

File: libnethack/src/shknam.c [hunk 1]
```diff
@@ -271,7 +235,7 @@ nameshk(struct monst *shk, const char *const *nlp, struct level *lev)
                     continue;
                 continue;       /* next `trycnt' iteration */
             } else {
-                shname = shk->female ? "Lucrezia" : "Dirk";
+                shname = shk->female ? "Paul Mezzanini" : "Matt Campbell";
```
Comment (rossdylan) [inline libnethack/src/shknam.c]: Shouldn't one of these be a female name?

Comment (dgonyeo) [inline libnethack/src/shknam.c]: Shush you. 

If you have a good candidate for a female that can occur often let me know, but until that happens Paul is female.

**Pull request merged**